src/app/routes/connector-ui/policy-definition-page/new-policy-dialog/new-policy-dialog.component.html

```diff
@@ -25,27 +25,67 @@ <h1 mat-dialog-title>Create New Policy</h1>
         </mat-option>
       </mat-select>
     </mat-form-field>
+    <!-- Time Interval Type -->
+    <ng-container *ngIf="form.policyType === 'Time-Period-Restricted'">
+      <div class="w-full flex flex-row space-x-[20px]" mat-dialog-actions>
+        <label>Date Selection Criteria:</label>
+        <mat-checkbox
+          color="accent"
+          [formControl]="form.group.controls.rangeIsOpenEnded"
+          >Use open-ended start date
```

Comment:
the start date can't be "open ended", it's just a point in time.

the interval is open ended.

so the text should be something along the lines:
- Use open-ended date interval
- Open Ended